templates/targets/x86_64/generic/flake.nix

```diff
@@ -83,7 +83,12 @@
             }
           ];
         };
-        packages.x86_64-linux.PROJ_NAME-ghaf-debug = self.nixosConfigurations.PROJ_NAME-ghaf-debug.config.system.build.${self.nixosConfigurations.PROJ_NAME-ghaf-debug.config.formatAttr};
+        packages.x86_64-linux.PROJ_NAME-ghaf-debug = let
+          hostConfiguration = self.nixosConfigurations.PROJ_NAME-ghaf-debug.config;
+          formatModules = nixos-generators.nixosModules.raw-efi;
+          config = (hostConfiguration.extendModules {modules = [formatModule];}).config;
```

Comment:
```
$ nix flake update
error: undefined variable 'formatModule'

       at /nix/store/c7dxv7d7q4gc8bbskf38h9si87gf1f1q-source/flake.nix:89:65:

           88|           formatModules = nixos-generators.nixosModules.raw-efi;
           89|           config = (hostConfiguration.extendModules {modules = [formatModule];}).config;
             |                                                                 ^
           90|         in
```

Reply:
The typo (missing `s`) was easily fixed but please fix and test running the commands yourself.